lib/pg_search_scope/model_helper.rb

```diff
@@ -87,7 +87,7 @@ def search_scope_for *column_names
             search_scope = search_scope.select("#{rank} #{scope_name}_rank")
           end
 
-          search_scope.where("#{tsvector} @@ #{tsquery}").order("#{rank} DESC")
+          search_scope.where(Arel.sql("#{tsvector} @@ #{tsquery}")).order(Arel.sql("#{rank} DESC"))
```

Comment:
The fix is needed only inside `order` method call.
Please remove `Arel.sql` from `where` method call.